thirdeye/thirdeye-frontend/app/pods/manage/alerts/performance/template.hbs

```diff
@@ -0,0 +1,121 @@
+<div class="manage-alert-tune">
+
+  <div class="te-content-block">
+    <h4 class="te-alert-page__subtitle">Alert Performance by Application Group</h4>
+
+    <a class="te-pill-selectors__side-link te-pill-selectors__side-link--{{if viewTotals "active" "inactive"}}" {{action (toggle "viewTotals" this)}}>
+      View total values
+    </a>
+    <a class="te-pill-selectors__side-link te-pill-selectors__side-link--{{if viewTotals "inactive" "active"}}" {{action (toggle "viewTotals" this)}}>
+      View average values
+    </a>
+
+    {{!-- Alert anomaly table --}}
+    <table class="te-anomaly-table te-anomaly-table--performance">
+      <thead>
+        <tr class="te-anomaly-table__row te-anomaly-table__head">
+         <th class="te-anomaly-table__cell-head">
+            <a class="te-anomaly-table__cell-link" {{action "toggleSortDirection" "name"}}>
+              Appp
+              <i class="te-anomaly-table__glyph te-anomaly-table__glyph--small glyphicon {{if sortColumnNameUp "glyphicon-menu-up" "glyphicon-menu-down"}}"></i>
+            </a>
+           </th>
+           <th class="te-anomaly-table__cell-head">
+            <a class="te-anomaly-table__cell-link" {{action "toggleSortDirection" "alert"}}>
+              Total Alerts
+              <i class="te-anomaly-table__glyph te-anomaly-table__glyph--small glyphicon {{if sortColumnAlertUp "glyphicon-menu-up" "glyphicon-menu-down"}}"></i>
+            </a>
+           </th>
+           <th class="te-anomaly-table__cell-head">
+            <a class="te-anomaly-table__cell-link" {{action "toggleSortDirection" "anomaly"}}>
+              Anomalies
+              <i class="te-anomaly-table__glyph te-anomaly-table__glyph--small glyphicon {{if sortColumnAnomalyUp "glyphicon-menu-up" "glyphicon-menu-down"}}"></i>
```

Comment:
Here as well. Let's do this for the rest of the PR.

Reply:
done